src/table.js

```diff
@@ -289,8 +288,9 @@ export default class Table {
 
   setCellWidth({row, column, adjustedWidth = 0, defaultWidth = 1}) {
     const cell = this.getCell(row, column);
-    const width = parseFloat(cell.dataset.width) || defaultWidth;
-    cell.dataset.width = Math.max(width + adjustedWidth, 0.1);
+    const width = parseFloat(cell.dataset.width) || parseFloat(defaultWidth);
+
+    cell.dataset.width = Math.max(width + parseFloat(adjustedWidth), 0.1);
```

Comment:
Have to parseFloat, as it seems like this may sometimes come out as strings and would give the wrong value.